listenbrainz/webserver/templates/user/scraper.js

```diff
@@ -428,7 +431,16 @@ function updateLatestImportTimeOnLB() {
     xhr.timeout = 10 * 1000; // 10 seconds
     xhr.onload = function(content) {
         if (this.status == 200) {
-            updateMessage("<i class='fa fa-check'></i> Import finished<br><span><a href={{ url_for('user.profile', user_name = user_name) }}>Close and go to your ListenBrainz profile</a></span><br><span style='font-size:8pt'>Thank you for using ListenBrainz</span>");
+            var final_msg = "<i class='fa fa-check'></i> Import finished<br>";
+            final_msg += "<span><a href={{ url_for('user.profile', user_name = user_name) }}>Close and go to your ListenBrainz profile</a></span><br>";
+            final_msg += "<span style='font-size:8pt'>Successfully submitted " + countReceived + " listens to ListenBrainz. Please note that some of these listens might be duplicates leading to a lower listen count on LB.";
+            if (playCount != -1 && countReceived != playCount) {
+                final_msg += " This number is different from the " + playCount +
```

Comment:
It would be nice to make this in a bold font in red, so that people have a bit more of a chance to see it.